python/vyos/ifconfig/interface.py

```diff
@@ -549,6 +612,9 @@ def set_netns(self, netns):
         >>> Interface('dum0').set_netns('foo')
         """
 
+        # Check if interface realy exists in namespace
+        if get_interface_namespace(self.ifname) != None:
```

Comment:
This would be faster and should achieve the same thing
```suggestion
        if _interface_exists_in_netns(self.ifname, netns):
```